src/OrbitQt/orbitmainwindow.cpp

```diff
@@ -1065,35 +1064,6 @@ void OrbitMainWindow::OnTimerSelectionChanged(const orbit_client_protos::TimerIn
   ui->liveFunctions->OnRowSelected(selected_row);
 }
 
-void OrbitMainWindow::on_actionSave_Capture_triggered() {
-  ShowCaptureOnSaveWarningIfNeeded();
```

Comment:
I think this is the only use of this method and as such it can be removed too. I might be wrong.

Reply:
right - removed.